provisioning/device/src/Microsoft.Azure.Devices.Provisioning.Client.csproj

```diff
@@ -64,10 +64,11 @@
     <PackageReference Include="Azure.Core" Version="1.26.0" />
     <PackageReference Include="Microsoft.Azure.Amqp" Version="2.5.12" />
     <PackageReference Include="MQTTnet" Version="4.1.2.350" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Not 13.02?

Reply:
I can update it to 13.0.2. I chose 13.0.1 for consistency only since other SDKs use this version.